code/drasil-printers/lib/Language/Drasil/Plain/Print.hs

```diff
@@ -30,10 +29,6 @@ data SingleLine = OneLine | MultiLine
 exprDoc :: PrintingInformation -> SingleLine -> L.Expr -> Doc
 exprDoc pinfo sl e = pExprDoc sl (expr e pinfo)
 
--- | Create code expressions for a document in 'Doc' format.
-codeExprDoc :: PrintingInformation -> SingleLine -> C.CodeExpr -> Doc
-codeExprDoc pinfo sl e = pExprDoc sl (codeExpr e pinfo)
```

Comment:
This function is used by `drasil-code` for a single purpose: rendering `CodeExpr`-based quantity constraints into human-readable text for outputting input value constraint violation messages (see `stable/` changes).

Reply:
This seems like reasonable functionality - where does it belong? And maybe it should be in two stages (i.e. `codeExprDoc` would get fed the results of `codeExpr e pinfo` so that `codeExpr` would not be needed here).